client/pages/Home.tsx

```diff
@@ -1,7 +1,19 @@
 import React from "react";
-import { View, Text, Button, StyleSheet, Pressable } from 'react-native';
+import { View, Text, Button, StyleSheet, Pressable, TouchableOpacity } from 'react-native';
+import { useNavigation } from '@react-navigation/core'
+
+import { auth } from "../configs/firebase";
 
 const Home = ({navigation}) => {
+    // const navigation = useNavigation();
```

Comment:
Does this overlap with Anita's pull-request? https://github.com/jonramm/Bike_Kollective/pull/11